cli/src/main/kotlin/org/jetbrains/research/pynose/headless/Util.kt

```diff
@@ -0,0 +1,66 @@
+package org.jetbrains.research.pynose.headless
+
+import com.jetbrains.python.inspections.PyInspection
+import org.jetbrains.research.pynose.plugin.inspections.pytest.DuplicateAssertionTestSmellPytestInspection
+import org.jetbrains.research.pynose.plugin.inspections.pytest.RedundantAssertionTestSmellPytestInspection
+import org.jetbrains.research.pynose.plugin.inspections.pytest.disabled.*
+import org.jetbrains.research.pynose.plugin.inspections.unittest.*
+import org.jetbrains.research.pynose.plugin.inspections.unittest.disabled.*
+import org.jetbrains.research.pynose.plugin.inspections.universal.*
+
+object Util {
+    fun getPytestInspectionsFunctionLevel(): Map<PyInspection, String> {
+        return mapOf(
+            Pair(ConditionalTestLogicTestSmellInspection(), "Conditional test logic"),
```

Comment:
Let's declare inspection names as properties for each inspection

Reply:
could you please clarify what you meant here?
adding name property to each inspection class or something else?